### PR TITLE
Filter out automated commits

### PR DESCRIPTION
**Pull request by akaRem**
It could be nicer to filter commits based on bot name, but then we need to reconfigure these pipelines everywhere to pass bot name.

But we have a convention to prefix `[AUTOMATED]` commits.. Which should work as well..

### CODE REVIEW

**Comment by akaRem** [comment]
Please review with ignored whitespace changes.

**Review by guilhermeassad — CHANGES_REQUESTED** [review]
Please check my comment.

File: .github/workflows/terraform-detect-drift-full-reusable.yml [hunk 1]
```diff
@@ -78,15 +80,18 @@ jobs:
...
 
           # echo the JSON list to $GITHUB_OUTPUT
           echo "environments=$json" >> $GITHUB_OUTPUT
       - name: Get last commit
         id: last-commit
         run: |
           #!/bin/bash
-          echo "last_commit=$(git log --pretty=format:"%an <%ae> - %cd - %s" --date=short | head -1)" >> $GITHUB_OUTPUT
+          # get short log messages excluding lines where "[AUTOMATED]" is present, because it is a signature of bot..
+          last_commit="$(git log --pretty=format:"%an <%ae> - %cd - %s" --date=short | grep -vi ' - \[AUTOMATED\] ' | head -1)"
```
[the remark this answers is not in the document]
Comment (akaRem) [reply .github/workflows/terraform-detect-drift-full-reusable.yml]: Fixed

**Review by guilhermeassad — APPROVED** [review]
Looks good

**Pull request merged**